src/pages/tutorials/getting-started-with-spring-boot.md

```diff
@@ -324,84 +413,38 @@ type Query {
 ...
 ```
 
-## Author DataFetcher
+### Author DataFetcher
+```java
+public DataFetcher getAuthorDataFetcher() {
```

Comment:
I think that is outdated, we use not annotated methods?

Reply:
As discussed I'll trim down the tutorial to just enough to get started